webroot/js/inject-iframe.js

```diff
@@ -101,6 +101,12 @@ if (elem) {
       doc.addEventListener('DOMContentLoaded', onReady, false);
       doc.addEventListener('DOMContentLoaded', proxyAjaxOpen, false);
       doc.addEventListener('DOMContentLoaded', proxyAjaxSend, false);
+
+      // Add support for turbo DOMContentLoaded alternative
+      // see https://turbo.hotwired.dev/reference/events#turbo%3Aload
+      doc.addEventListener('turbo:load', onReady, false);
```

Comment:
This will close the toolbar and attach duplicate message handlers. I don't think that's what we want.